Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM debian
```

Comment:
having a dockerfile is nice, but it would require some documentation as to how it is intended to be used ...